windows/SettingsManager.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using Windows.Storage;
+
+namespace CodePush.ReactNative
+{
+    internal class SettingsManager
+    {
+        private static ApplicationDataContainer GetCodePushSettings()
+        {
+            return ApplicationData.Current.LocalSettings.CreateContainer(CodePushConstants.CodePushPreferences, ApplicationDataCreateDisposition.Always);
+        }
```

Comment:
I wonder whether it would be nicer to have a static constructor that initialized a private variable, that all other methods in this class use internally. The static constructor won't get called until one of its classes methods are actually called, so you gain the same benefits of deferring this code until it's necessary, but the code that consumes it would arguably be a little cleaner.